apis/datasciencecluster/v1/datasciencecluster_types.go

```diff
@@ -46,7 +46,7 @@ type Components struct {
 	Workbenches workbenches.Workbenches `json:"workbenches,omitempty"`
 
 	// ModelMeshServing component configuration.
-	// Require CodeFlare Operator to be installed before enable component
+	// If CodeFlare Operator is not already installed, it'll be installed on the fly.
```

Comment:
i think the old comments is wrong, should be removed, modelmesh does not need codeflare as pre-req

Reply:
done